lib/services/ngsi/subscriptionService.js

```diff
@@ -133,18 +186,97 @@ function subscribe(device, triggers, content, callback) {
             config.getConfig().contextBroker.port + '/v1/subscribeContext';
```

Comment:
Raised during integration e2e testing, it seems lines L185-L186 are affected by the recent change in CB URL configuration and should be changed by:

```
options.uri = config.getConfig().contextBroker.url + '/v1/subscribeContext';
```

I guess that a similar change should be applied to NGSIv2 branch logic, in L246-247.

Reply:
It would be great to include also a unit test to reproduce the problem and ensure the fix (although I'm not sure if this could be easy...).

Reply:
Fixed in ff9c387503b4acff613369053ce50f79a4c3f12e